src/stories/components/Highlight.stories.mdx

```diff
@@ -72,20 +72,20 @@ export const docsDecorator = () => {
 
 # Highlight
 
-### Highlight são componentes utilizados destacar palavras ou frases importantes em um determinado contexto.
+### Highlights são componentes utilizados destacar palavras ou frases importantes em um determinado contexto.
 ---
 <br />
 
 ## Recomendamos o uso de Highlights quando:
-- Você deseja guiar a atenção do usuário para palavras ou frases espefícias na interface (nesses cenários Highlights animados podem ser ainda mais efetivos que os comuns)
-- Você está implementando alguma feature de <i>diff</i> e precisa indicar elementos que foram substituídos por outros
+- Se deseja guiar a atenção do usuário para palavras ou frases específicas na interface (nesses cenários, Highlights animados podem ser ainda mais efetivos que os comuns).
+- Estiver implementando alguma feature de diff e precisar indicar elementos que foram substituídos por outros.
```

Comment:
O itálico na versão anterior era pra indicar o estrangeirismo. Tu acha que a gente precisa tirar?

Reply:
Esqueci de novo.